ci/test/utils/fetch-merge-target.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# This requires BUILDKITE_REPO_REF to be set.
+
+set -euo pipefail
+
+. misc/shlib/shlib.bash
+
+ci_collapsed_heading "Configure git"
+run git config --global user.email "buildkite@materialize.com"
```

Comment:
Running this locally will be annoying. Could surround this with a check of whether we are inside BUILDKITE.

Reply:
There is a `BUILDKITE="true"` environment variable if we are inside of buildkite. See https://buildkite.com/docs/pipelines/environment-variables

Reply:
✅ https://github.com/MaterializeInc/materialize/pull/19157/commits/bf249db196ec9acb015ad00660ee3f90925f7337